### ISSUE TITLE
Fix: accessibility on bus page's info button

### ISSUE THREAD

**Issue by hiromon0125** [hiromon0125/ritchie-bus-schedule#112]
**Have you searched in [Issues](https://github.com/hiromon0125/ritchie-bus-schedule/issues) for a similar bug report?**
y

**Describe the bug**
The alt message is missing on the info icon on the bus top list. Same for stop page.

**To Reproduce**
1. Go to '/bus/1' or 'stop/1'
2. Inspect the info button on the bus stop list widget.

**Expected behavior**
The button should have a proper aria-label to let screen readers read as a info button instead of "button". I think this aria-label could also just be the text content of the popup that the button has.

**Screenshots**
![Image](https://github.com/user-attachments/assets/1273106c-77ab-46b5-9db2-9cf9cdb1e722)

**Device (please complete the following information):**
 - all browser

**Additional context**
https://pagespeed.web.dev/analysis/https-www-rit-bus-app-bus-1/icbenpb885?hl=en-US&form_factor=mobile

**Comment by TylerAmey** [comment]
@hiromon0125 I think I'll pull this down to work on, does that work for you?

**Comment by hiromon0125** [comment]
sounds good. Let me know if you need help setting stuff up.
